package.json

```diff
@@ -13,7 +13,7 @@
     "prepare": "npm run -s build",
     "prepare:babel": "babel --presets env src/*.js -d dist && npm t",
     "lint": "eslint src",
-    "test": "npm run -s lint && npm run -s build && jest",
+    "test": "npm run -s lint && npm run -s build && jest --env=node",
```

Comment:
Without this tests were failing for me with cryptic jsdom errors. I assume it is okay to keep as the tests should run in node env either way

Reply:
It also seems that this is why all these greenbot builds are failing right now 🤔 The same error I was getting:

```
    SecurityError: localStorage is not available for opaque origins
```